recipe/meta.yaml

```diff
@@ -30,9 +35,11 @@ test:
   commands:
```

Comment:
So the Python version used in the test phase is not required to match the Python version in the build phase. ( https://github.com/conda/conda-build/issues/835 ) Normally this would happen by making Python a run time dependency, but that is clearly not needed. This is how one can fix that without making Python a run time dependency.